### PR TITLE
refactor(seating-chart): use ScaledEmptyState for empty states

### PR DESCRIPTION
**Pull request by OPS-PIvers**
## Summary

- Replace the two hand-rolled empty states in the SeatingChart widget (`components/widgets/SeatingChart/Widget.tsx`) with the shared `ScaledEmptyState` primitive.
- The previous empty states used hardcoded Tailwind size classes (`w-12 h-12`, `text-sm`, `text-xs`) that violate the container-query scaling standard documented in `CLAUDE.md` and produced inconsistent styling vs. the other 15+ widgets already using `ScaledEmptyState`.
- Existing i18n keys (`widgets.seatingChart.emptyStateFreeform` / `emptyStateTemplate`), icons (`LayoutGrid`, `LayoutTemplate`), and the `pointer-events-none` overlay behavior are preserved exactly.
- Add a focused test suite for the SeatingChart empty states (6 cases).

## Why

Low-burden, high-reward polish. SeatingChart is one of the most visible widgets (default 600×500) and the fix is a pure find-and-replace onto an already-tested primitive — no new component design, no API surface changes, no data-flow changes.

## Test plan

- [x] `pnpm run type-check` — clean
- [x] `pnpm run lint` — zero errors, zero warnings
- [x] `pnpm run format:check` — clean
- [x] `pnpm run test tests/components/widgets/SeatingChart.test.tsx` — 6/6 pass
- [x] `pnpm run test` — full unit suite green (1183/1183 across 132 files)
- [ ] Manual smoke: add SeatingChart widget, toggle between Assign / Setup modes, resize small, confirm empty states scale and canvas still receives clicks (empty-state overlay is `pointer-events-none`)

### New test coverage (tests/components/widgets/SeatingChart.test.tsx)

1. Renders the `"Empty Classroom"` empty state in the default (non-setup) mode.
2. Renders the `"No Furniture"` + freeform subtitle in setup mode with `template: 'freeform'`.
3. Renders the `"No Furniture"` + template subtitle in setup mode with `template: 'rows'`.
4. Renders neither empty state when the `furniture` array is non-empty.
5. Empty-state overlay has `pointer-events-none` so it never blocks canvas interactions.
6. Guard test: title/subtitle nodes are NOT decorated with the legacy `text-sm`/`text-xs` classes, and the `ScaledEmptyState` shape (`uppercase tracking-widest` title) is present. Prevents the hand-rolled pattern from re-entering the file.

https://claude.ai/code/session_01Q8p3Cz7Jgbnpw1HgBu1atx

### CODE REVIEW

**Comment by OPS-PIvers** [comment]
/gemini review this PR thoroughly and leave comments for all issues from nits to critical.

File: components/widgets/SeatingChart/Widget.tsx [hunk 1]
```diff
@@ -956,28 +957,32 @@ export const SeatingChartWidget: React.FC<{ widget: WidgetData }> = ({
...
-                Switch to &quot;Setup&quot; to arrange furniture.
-              </p>
+            <div
+              data-testid="seating-chart-empty-assign"
+              className="absolute inset-0 pointer-events-none"
+            >
+              <ScaledEmptyState
+                icon={LayoutGrid}
+                title="Empty Classroom"
+                subtitle='Switch to "Setup" to arrange furniture.'
```
[the remark this answers is not in the document]
Comment (OPS-PIvers) [reply components/widgets/SeatingChart/Widget.tsx]: Already addressed in this PR's diff. The hardcoded `"Empty Classroom"` / `Switch to "Setup" to arrange furniture.` strings and the setup-mode `"No Furniture"` title were moved into `locales/en.json` under new keys `widgets.seatingChart.emptyStateAssignTitle`, `emptyStateAssignSubtitle`, and `emptyStateSetupTitle`, and the component now routes all four strings through `t()`.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Review by OPS-PIvers — COMMENTED** [review]
## Automated Code Review

**Date:** 2026-04-17

### Summary
Replaces the two hand-rolled empty states inside `SeatingChart/Widget.tsx` with the shared `ScaledEmptyState` primitive, moves the previously hardcoded `"Empty Classroom"` / `"No Furniture"` strings into `locales/en.json` under new keys (`emptyStateAssignTitle`, `emptyStateAssignSubtitle`, `emptyStateSetupTitle`), and adds a focused 6-case test suite that also guards against the legacy `text-sm` / `text-xs` pattern regressing. Tight, well-scoped polish PR.

### Code Quality
✓ Clean. Preserves `pointer-events-none` so the overlay doesn't block the setup canvas. Uses i18n via `t()` for all four strings. No stray `console.log` / dead code.

### Widget Standards
✓ Removes the exact anti-pattern called out in `CLAUDE.md` ("never hardcoded Tailwind size classes like `text-sm`, `text-xs`, `w-12 h-12`") from front-face content. The new empty states delegate sizing to `ScaledEmptyState`, which is `cqmin`-based.
- Minor nit: the `data-testid` hooks (`seating-chart-empty-assign` / `-setup`) are unusual for this codebase, but harmless and justified by the new guard tests — they give the test a stable anchor without asserting on implementation details.

### Admin Config Standards
✓ Not applicable (no admin surface touched).

### Firestore & Security
✓ No rules / data-model / auth changes.

### Regression Risk
✓ Low. Scope is exactly 3 files.
- **Minor i18n gap (pre-existing, not introduced here):** the new keys were only added to `locales/en.json`; `de.json` / `es.json` / `fr.json` don't have them. However the existing sibling keys (`emptyStateFreeform`, `emptyStateTemplate`) are also missing from those files, so this PR doesn't regress the situation — it just doesn't fix it. Worth tracking separately.
- `skipScaling: true` remains correct for SeatingChart; no registry or defaults change needed.

### Missing Tests
✓ Good coverage added — 6 cases covering both modes, both templates, furniture-present guard, `pointer-events-none` contract, and a regression guard that asserts the legacy `text-sm`/`text-xs` classes are absent. Nicely targeted.

### Overall
**Merge readiness:** Ready with minor notes.
Pure find-and-replace onto an already-tested primitive with solid test coverage; the only open thread (i18n for these strings) is itself addressed by this PR's own diff — reply posted. Un-drafting looks reasonable once non-English locales are either backfilled here or tracked in a follow-up.

---
_This review was generated automatically. All findings should be verified by a human before merging._

---
_Generated by [Claude Code](https://claude.ai/code)_
